metadata_service/proxy/neo4j_proxy.py

```diff
@@ -816,6 +816,46 @@ def get_latest_updated_ts(self) -> Optional[int]:
         else:
             return None
 
+    @timer_with_counter
+    def get_statistics(self) -> Dict[str, Any]:
```

Comment:
currently it only include table stats, do we want to extend it for dashboard as well?

Reply:
We currently use in our Amundsen implementation just the tables, so that's why I didn't add the dashboards statistics, which I guess can be easily extended to in a follow-up MR if needed by another team.